config/sanitizer/README.md

```diff
@@ -45,15 +31,9 @@ This is a collection of quite useful scripts that expand the possibilities for b
   - [clang-format](#clang-format)
   - [cmake-format](#cmake-format)
 
-## C++ Standards [`c++-standards.cmake`](c++-standards.cmake)
-
-Using the functions `cxx_11()`, `cxx_14()`, `cxx_17()` or `cxx_20()` this adds the appropriate flags for both unix and MSVC compilers, even for those before 3.11 with improper support.
-
-These obviously force the standard to be required, and also disables compiler-specific extensions, ie `--std=gnu++11`. This helps to prevent fragmenting the code base with items not available elsewhere, adhering to the agreed C++ standards only.
-
 ## Sanitizer Builds [`sanitizers.cmake`](sanitizers.cmake)
 
-Sanitizers are tools that perform checks during a program's runtime and return issues, and as such, along with unit testing, code coverage and static analysis, are another tool to add to the programmer's toolbox. And, of course, like the previous tools, they are simple to add to any project using CMake, allowing any project and developer to quickly and easily use them.
+Sanitizers are tools that perform checks during a program’s runtime and returns issues, and as such, along with unit testing, code coverage and static analysis, is another tool to add to the programmers toolbox. And of course, like the previous tools, are tragically simple to add into any project using CMake, allowing any project and developer to quickly and easily use.
```

Comment:
return and are were correct;  the subject  is either tools or sanitizers, and both are plural.  Also "tragically simple" seems a little odd, maybe "trivially simple" or just simple would be better? 
Also, unless there's a reason I missed, it might be best to more clearly identify the previous tools that are simple to add into any project.